dashboard/src/app/workspaces/workspace-details/workspace-details.controller.ts

```diff
@@ -588,8 +588,8 @@ export class WorkspaceDetailsController {
    * Checks whether "plugins" are enabled in workspace config.
    */
   isPluginsEnabled(): boolean {
-    let editor = this.workspaceDetails.config.attributes.editor || '';
-    let plugins = this.workspaceDetails.config.attributes.plugins || '';
+    let editor = this.workspaceDetails.config.attributes ? this.workspaceDetails.config.attributes.editor : '';
```

Comment:
shouldn't it be an empty array instead of '' ? (same for the next line)

Reply:
that whole method may as well be removed since it's not used

Reply:
removed the method